recipe/session/utils.go

```diff
@@ -380,7 +384,22 @@ func getRequiredClaimValidators(
 		return nil, err
 	}
 	if overrideGlobalClaimValidators != nil {
-		globalClaimValidators = overrideGlobalClaimValidators(globalClaimValidators, sessionContainer, userContext)
+		globalClaimValidators, err = overrideGlobalClaimValidators(globalClaimValidators, sessionContainer, userContext)
+		if err != nil {
+			return nil, err
+		}
 	}
 	return globalClaimValidators, nil
 }
+
+type fakeRes struct{}
+
+func (f fakeRes) Header() http.Header {
+	return http.Header{}
+}
+
+func (f fakeRes) Write(body []byte) (int, error) {
+	return len(body), nil
+}
+
+func (f fakeRes) WriteHeader(statusCode int) {}
```

Comment:
What is this for?

Reply:
fake http response object for creating session in tests:
<img width="846" alt="image" src="https://user-images.githubusercontent.com/650429/188549785-5db7bb5f-d46b-464c-abc4-f267905ccf51.png">

moving it to testingUtils.go instead of utils.go